### PR TITLE
19384 PNPM replacement

### PR DESCRIPTION
**Pull request by eve-git**
*Issue #:* /bcgov/entity###
https://github.com/bcgov/entity/issues/19384

*Description of changes:*
PNPM replacement

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-758-wz8vlgsi.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-758-wz8vlgsi.web.app

File: Makefile
Comment (pwei1018) [inline Makefile:1]: This file can be removed.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-758-wz8vlgsi.web.app

**Comment by eve-git** [comment]
There are three tasks with errors. @pwei1018 @bolyachevets, is this acceptable? Any suggestions?

**Comment by bolyachevets** [comment]
> There are three tasks with errors. @pwei1018 @bolyachevets, is this acceptable? Any suggestions?

the steps are failing, because it is still using the old workflow currently in namerequest repo (which calls Makefile etc.), the new workflow Patrick is working on is referenced here https://github.com/bcgov/namerequest/pull/758

**Comment by eve-git** [comment]
**Please hold on merge till sprint 21.2 released by April 30th noon**

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-758-wz8vlgsi.web.app

File: package.json
Comment (bolyachevets) [inline package.json:1]: need to add 

"build-check": "node --max_old_space_size=8000 node_modules/@vue/cli-service/bin/vue-cli-service.js build", in the scripts section of package.json for the workflow to work

**Comment by seeker25** [comment]
Thanks for this, book marking for auth-web

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-758-wz8vlgsi.web.app

**Review by bolyachevets — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,32 +6,32 @@
...
 ## Project setup
 ```
-npm install
+pnpm install
 ```
 
 ### Compiles and hot-reloads for development
 ```
-npm run serve
+pnpm run serve
```
Comment (severinbeauvais) [inline README.md:14]: You need to let people know about this change.

And also let them know they need to do `npm i -g pnpm` before they can build this project.

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -0,0 +1,14246 @@
...
+        version: 6.12.1
+      quill:
+        specifier: ^2.0.0
+        version: 2.0.0
+      remove-accents:
+        specifier: ^0.4.2
+        version: 0.4.4
+      sbc-common-components:
+        specifier: 3.0.8
+        version: 3.0.8
```
Comment (severinbeauvais) [inline pnpm-lock.yaml:76]: For some reason, this is out of date. (The package file specifies 3.0.12.)

Anyway, I updated this in PR #764.

**Pull request merged**